build.rs

```diff
@@ -45,16 +42,15 @@ fn main() {
         println!("cargo:rustc-link-lib=c++");
     }
 
-    println!("cargo:rerun-if-changed=wrapper.h");
-
```

Comment:
This is no longer needed after the update to the latest version of bindgen.
See https://docs.rs/bindgen/latest/src/bindgen/lib.rs.html#1252-1277